x-pack/plugins/ml/server/models/data_recognizer/modules/security_network/manifest.json

```diff
@@ -36,22 +36,22 @@
   ],
   "datafeeds": [
     {
-      "id": "datafeed_high_count_by_destination_country",
+      "id": "datafeed-high_count_by_destination_country",
```

Comment:
Good catch here. Just curious: what was the manifestation of this typo?

Reply:
@rylnd this was missed in the reviews of https://github.com/elastic/kibana/pull/96480. The convention we follow for IDs in the ML module manifest is to use underscores in the job IDs, and the datafeed ID in the format `datafeed-<job id>` (with a hyphen separating `datafeed` from the job ID part). One [commit](https://github.com/elastic/kibana/pull/96480/commits/f08de924ea49891febfe0001556eb426d993622a) in the original PR corrected hyphens in the job IDs to underscores, but we just missed that the datafeed ID didn't use the initial hyphen.